.vscode/launch.json

```diff
@@ -40,6 +40,33 @@
 				"${workspaceFolder}/../klighd-vscode/applications/klighd-vscode/dist/**/*.js"
             ],
         },
+        {
+            "name": "Launch VS Code Extension (STPA)",
```

Comment:
I think the name should be something like: Launch VS Code Extension with PASTA/STPA